File: Python/9. Palindrome Number.py
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False
        sum1, sum2 = 0, x
        while x:
            sum1 = sum1 * 10 + x%10
            x //= 10
        return sum1 == sum2  

File: C++/9. Palindrome Number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0)
            return false;
        int sum1 = 0, sum2 = x;
        while(x)
        {
            sum1 = sum1*10 + x%10;
            x /= 10;
        }
        return sum1 == sum2;
    }
};